.gitignore

```diff
@@ -2,10 +2,11 @@
 **/*.idea/
 
 ## Build Artifacts
+**/node_modules
 **/target
 wasm/Cargo.lock
 **/package-lock.json
-**/build
+wasm/build.rs
```

Comment:
I think the old `**/build` was to ignore projects created with the aleo CLI so we may want to keep it.

follow up question: when is `wasm/build.rs` generated?

Reply:
I'm actually unsure when this got change, it might've been an overzealous IDE-driven bulk renaming. Reverted!